CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.6.0 (March 23, 2022)
```

Comment:
I'm not sure if it matters for the new release tooling, but I usually see an Unreleased section at the top with these ([example](https://github.com/hashicorp/vault-csi-provider/pull/130/files)):

```suggestion
## Unreleased

## 0.6.0 (March 23, 2022)
```

Reply:
Thanks!